packages/p2p/src/components/buy-sell/buy-sell-table.jsx

```diff
@@ -26,10 +26,11 @@ const BuySellRowRendererComponent = row_props => {
 const BuySellRowRenderer = observer(BuySellRowRendererComponent);
 
 const BuySellTable = ({ onScroll }) => {
-    const { buy_sell_store, general_store } = useStores();
+    const { buy_sell_store, general_store, my_profile_store } = useStores();
 
     React.useEffect(
         () => {
+            my_profile_store.getPaymentMethodsList();
```

Comment:
There was an issue when switching to sell ads in buy-sell page, the payment methods were not listed for each sell ad that has payment methods since `my_profile_store,available_payment_methods` is still empty and not yet initialized, thus the ads with payment methods were filtered out of their payment methods